include/tvm/target/target.h

```diff
@@ -122,14 +125,28 @@ class Target : public ObjectRef {
   TVM_DLL explicit Target(std::nullptr_t) { data_ = nullptr; }
   /*!
    * \brief Construct a Target given a string
-   * \param tag_or_config_or_target_str the string to parse
+   * \param tag_or_config_or_target_str the string to parse for target
    */
   TVM_DLL explicit Target(const String& tag_or_config_or_target_str);
+  /*!
+   * \brief Construct a Target given a string
+   * \param tag_or_config_or_target_str the string to parse for target
+   * \param host_tag_or_config_or_host_str the string to parse for target host
+   */
+  TVM_DLL explicit Target(const String& tag_or_config_or_target_str,
+                          const String& host_tag_or_config_or_host_str);
   /*!
    * \brief Construct a Target using a JSON-like configuration
-   * \param config The JSON-like configuration
+   * \param config The JSON-like configuration for target
    */
   TVM_DLL explicit Target(const Map<String, ObjectRef>& config);
+  /*!
+   * \brief Construct a Target using a JSON-like configuration
+   * \param config The JSON-like configuration for target
+   * \param host_config The JSON-like configuration for target host
+   */
+  TVM_DLL explicit Target(const Map<String, ObjectRef>& config,
+                          const Map<String, ObjectRef>& host_config);
```

Comment:
nit: No need to provide these two constructors - C++ users can assemble them up using Target(Target(dict_a), Target(dict_b)).

Reply:
That's right. I have removed the redundant code.